themes/theme-gmd/pages/Scanner/index.jsx

```diff
@@ -42,17 +42,15 @@ class ScannerView extends PureComponent {
       <View background={backgroundColor}>
         <RouteContext.Consumer>
           {({ query: { scope = SCANNER_SCOPE_DEFAULT, type = SCANNER_TYPE_BARCODE } = {} }) => (
-            <Fragment>
+            <ScannerContainer
+              scope={scope}
+              type={type}
+              scannerDidOpen={this.removeBackground}
+              scannerDidClose={this.resetBackground}
+            >
               <BackBar title="titles.scanner" right={null} />
```

Comment:
Same here: the children of the container are only rendered when the scanner iss supported and activated by config.